benchmarks/benchmarks/stats.py

```diff
@@ -19,4 +18,84 @@ def setup(self, *args):
     def time_anderson_ksamp(self):
         with warnings.catch_warnings():
             warnings.simplefilter('ignore', UserWarning)
-            anderson_ksamp(self.rand)
+            stats.anderson_ksamp(self.rand)
+
+
+class CorrelationFunctions(Benchmark):
+    param_names = ['alternative']
+    params = [
+        ['two-sided', 'less', 'greater']
+    ]
+
+    def setup(self, mode):
+        a = np.random.rand(2,2)*10
```

Comment:
Oh, didn't know that. Thanks for the info. I suppose the following change now adheres to PEP8 ?
 
    a = np.random.rand(2,2) * 10